backend/src/emissions/schema.py

```diff
@@ -840,6 +846,57 @@ def mutate(root, info, input=None):
             return SetWorkingGroup(user=user, success=success, errors=e)
 
 
+
+
+class AnswerJoinRequest(graphene.Mutation):
```

Comment:
I am missing an endpoint that lists pending join requests for a certain working group. Do we have something like this planned? Is this something for another PR and an extra issue?

Reply:
good idea, I can add an endpoint in this PR.

Reply:
done.